pkg/api/networkservice/connection.proto

```diff
@@ -25,53 +25,54 @@ import "ptypes/timestamp/timestamp.proto";
 import "connectioncontext.proto";
 
 message Mechanism {
-  string cls = 1;
-  string type = 2;
-  map<string, string> parameters = 3;
+    string cls = 1;
+    string type = 2;
+    map<string, string> parameters = 3;
 }
 
 enum State {
-  UP = 0;
-  DOWN = 1;
+    UP = 0;
+    DOWN = 1;
 }
 
 message PathSegment {
-  string name      = 1;
-  string id        = 2;
-  string token = 3;
-  google.protobuf.Timestamp expires = 4;
+    string name = 1;
+    string id = 2;
+    string token = 3;
+    google.protobuf.Timestamp expires = 4;
+    map<string, string> metrics = 5;
```

Comment:
Ah... looks like this is the one and only non-whitespace change.  Is there some sort of proto linter you are using?